.kokoro/populate_secrets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
```

Comment:
Is it a script shared by multiple repositories (i.e. was it copied here, or was it written from scratch)? I'm asking mainly from the security perspective (if it has been tested before for a potential secrets "leaks", since it does some non-trivial stuff on them).

Reply:
This is the same secret loader in all the java client libraries (managed by templates). The gax repo does not share the same templates as many are related to maven builds and not gradle.